entrypoint.sh

```diff
@@ -1,46 +1,49 @@
 #!/bin/bash
+set -e
 
 # Set path
-echo '::group::Copying file from $WORKPATH to /tmp/gh-action'
+echo "::group::Copying file from $WORKPATH to /tmp/gh-action"
 WORKPATH=$GITHUB_WORKSPACE/$INPUT_PATH
 # Set path permision
 sudo -u builder mkdir /tmp/gh-action
-sudo -u builder cp -rfv $WORKPATH/.git /tmp/gh-action/.git
-sudo -u builder cp -fv $WORKPATH/PKGBUILD /tmp/gh-action/PKGBUILD
-cd /tmp/gh-action
-echo '::endgroup::'
+sudo -u builder cp -rfv "$WORKPATH"/.git /tmp/gh-action/.git
+sudo -u builder cp -fv "$WORKPATH"/PKGBUILD /tmp/gh-action/PKGBUILD
+cd /tmp/gh-action || exit
```

Comment:
In case /tmp/gh-action doesn't exist, the script will exit here with exit code 0. from the outside it will look like the action succeded, but it bailed out.
I would remove the ` || exit`

Reply:
Thank you for suggestion. I didn't know that.